knative-operator/pkg/common/eventing.go

```diff
@@ -17,6 +20,33 @@ func MutateEventing(ke *eventingv1alpha1.KnativeEventing, c client.Client) error
 	return nil
 }
 
+// eventingImagesFromEnviron overrides registry images
+func eventingImagesFromEnviron(ke *eventingv1alpha1.KnativeEventing, _ client.Client) error {
```

Comment:
I concur with @markusthoemmes that there is opportunity to factor some common code out here. Adding `eventing.go` in here makes me want to consider renaming `openshift.go` to `serving.go`. There is already a `util.go` in there that the common code for this function could go.

Reply:
Done. See
- https://github.com/openshift-knative/serverless-operator/pull/126/commits/cc97fa54e147a23eaf472bf9a20714289c82e945
- https://github.com/openshift-knative/serverless-operator/pull/126/commits/8f2eb99ed1caf09b1694d25c70a2d9eb3b1c87c1